python/cugraph/utilities/utils.py

```diff
@@ -173,6 +176,72 @@ def get_traversed_path_list(df, id):
     return answer
 
 
+def get_traversed_cost(df, source_col, dest_col, value_col):
+    """
+    Take the DataFrame result from a BFS or SSSP function call and sums
+    the given weights along the path to the starting vertex.
+
+    Input Parameters
+    ----------
+    df : cudf.DataFrame
+        The dataframe containing the results of a BFS or SSSP call
+    source_col : cudf.DataFrame
+        This cudf.Series wraps a gdf_column of size E (E: number of edges).
+        The gdf column contains the source index for each edge.
+        Source indices must be an integer type.
+    dest_col : cudf.Series
+        This cudf.Series wraps a gdf_column of size E (E: number of edges).
+        The gdf column contains the destination index for each edge.
+        Destination indices must be an integer type.
+    value_col : cudf.Series
+        This cudf.Series wraps a gdf_column of size E (E: number of edges).
+        The gdf column contains values associated with this edge.
+        Weight can be of any type
+
+    Returns
+    ---------
+    df : cudf.DataFrame
+        DataFrame containing two columns 'vertex' and 'info'.
```

Comment:
Should specify what the unreachable value is.

Reply:
Updated here: https://github.com/rapidsai/cugraph/pull/1376/commits/b6fbcec5e609b1b32c0d292dde9723afb8b0bb4e

Reply:
Thanks. However, unreachable should be infinite ` std::numeric_limits<vertex_t>::max()` not 0. The idea is that they do not get mixed with vertices that are free to go to. BFS/SSSP should do that already.